src/main/c/PageRequest.cpp

```diff
@@ -65,13 +65,10 @@ size_t PageRequest::getUintHeader(const std::string& name) const {
     }
     try {
         const auto val = std::stoi(iter->second);
+        return val < 0 ? 0u : val;
```

Comment:
Imho `std::max` brings over the message better: we want the maximum value of both of them.

But I'm nitpicking here.

Reply:
Commit updated :+1: